exercise3-2/src/camera_demo/src/main.cc

```diff
@@ -21,7 +22,7 @@ int main(int argc, char** argv) {
   RegisterThreadTracer(object_detector_tracer);
   RegisterThreadTracer(data_logger_tracer);
 
-  rclcpp::executors::MultiThreadedExecutor executor;
+  rclcpp::executors::MultiThreadedExecutor executor(rclcpp::ExecutorOptions(), 2);
```

Comment:
Hopefully all of this code can be explained to the attendees. Right now this looks pretty opaque to me.